Lib/test/test_gettext.py

```diff
@@ -37,6 +37,9 @@
 bmsgd2luayAoaW4gIm15IG90aGVyIGNvbnRleHQiKQB3aW5rIHdpbmsA
 '''
 
+# Invalid magic number ('ABCD' b64-encoded)
+GNU_MO_DATA_BAD_MAGIC_NUMBER = rb'\QUJDRA=='
```

Comment:
Any reason not to use `base64.b64encode("ABCD")`?

Reply:
All the other MO files were stored as base64 so I thought I'd do the same, but in this case I agree that it's more readable to just use `b64encode(b'ABCD')` directly since it's just the magic and nothing else. I updated the PR.